src/CommandBus/TacticianCommandBusAdapter.php

```diff
@@ -20,19 +20,19 @@
 
 namespace PrestaShop\Module\PrestashopCheckout\CommandBus;
 
-use League\Tactician\CommandBus;
+use PrestaShopBundle\CommandBus\MessengerCommandBus;
```

Comment:
Ah seems like you actually do use the `MessengerCommandBus` adpter from the core, my bad I was misled by the name of this class But since it's no longer based on Tactitian maybe it should be renamed as well (along with its factory)?